Source/Scene/Cesium3DTileFeatureTableResources.js

```diff
@@ -68,14 +64,13 @@ define([
                 // This is a reference to the binary
                 featureSize = defaultValue(featureSize, 1);
                 var typedArray = this.getTypedArrayForSemantic(semantic, byteOffset, componentType, this.featuresLength, featureSize);
-                var subArray = typedArray.subarray(featureId * featureSize, featureId * featureSize + featureSize);
-                if (featureSize === 1) {
-                    return subArray[0];
-                }
-                return subArray;
+                return typedArray.subarray(featureId * featureSize, featureId * featureSize + featureSize);
```

Comment:
I'm not sure if this is the best change. It makes more sense in `getGlobalProperty`, but having to access the returned value by `property[0]` all the time just seems weird.